modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -114,7 +114,7 @@ This folder stores the initial files loaded for a new user during webUI acceptan
 [[running-acceptance-tests]]
 === Preparing to Run Acceptance Tests
 
-This is a concise guide to running acceptance tests on ownCloud 10.0.
+This is a concise guide to running acceptance tests on ownCloud 10.
```

Comment:
This guide works for ownCloud 10.0 and 10.1 and I expect it not to change much for any ownCloud 10.*